modules/tumblr/Team1Report.md

```diff
@@ -0,0 +1,11 @@
+#Annotations of Underground Episode 1:
+
```

Comment:
The Television show, Underground, follows multiple story lines, involving field slaves, abolitionists, house slaves, and slave traders, plantation owners and more. These interactions not only provide us with insight in the brutal realities of the era, but allow the audience to witness the complex dynamics between individuals and groups of different backgrounds, races, and beliefs. 
The first episode begins with an action packed chase scene. Noah, a black man working on a plantation in a Georgia plantation, runs from vicious slave catchers armed with guns and dogs. Upon his capture, we experience the Unfortunately, he is caught, hurting his leg in the process. Before returning to the plantation, Noah and other runaway slaves are brought in shackles to a holding cell, where Noah discovers instructions which may lead to freedom. When he returns, he tells a few other men of his plan, still cautious as to whom he can trust.
The abolitionist is a man named John Hawks (a character fashioned from John _ Hawks, a real man). We first meet him on the steps of the courthouse, asserting that Dred Scott, a black man being prosecuted for running away, should receive proper representation in court. He argues that United States should not act as the oppressors’ forefathers by restricting freedom and instead, recognize the rights of all people in the nation, including slaves. In spite of the northern location, virtually all individuals within earshot of Hawks ignored his statement, alluding an unspoken tension between mainstream northern culture in the 1850s and abolitionist rhetoric many illustrations of slavery fail to acknowledge. Various other complications arise in his life, from his wife, Elizabeth, being unable to bear children and a request for his house to be apart of the Underground Railroad.
Additionally, there are the house slaves who are nearly all of lighter complexion, and mostly women. Although they are not regularly exposed to the grueling conditions and treatment field slaves receive, they are not safe from the wrath of the plantation owner, Tom Macon and his wife Suzanna. Interactions with the field slaves are limited, although some, like Rosalee, have a good relationship with field slaves, having delivered a child for one, and shared kind words with another. 
We also meet ___, who helps a runaway slave sneak past the slave traders. He claims that he helps people like her to create a better world for his younger son. _ appears to be a family man as well, sending money for his ailing wife at the Boston Hospital.